spec/lib/backup_restore/backuper_spec.rb

```diff
@@ -16,4 +16,25 @@
 
     expect(backuper.send(:get_parameterized_title)).to eq("coding-horror")
   end
+
+  describe '#notify_user' do
+    before { STDOUT.stubs(:write) }
+
+    it 'include upload' do
+      backuper = BackupRestore::Backuper.new(Discourse.system_user.id)
+      expect { backuper.send(:notify_user) }
+        .to change { Topic.private_messages.count }.by(1)
+        .and change { Upload.where(original_filename: "log.txt.zip").count }.by(1)
+    end
+
+    it 'includes upload error if cannot save upload' do
+      SiteSetting.max_export_file_size_kb = 1
+      SiteSetting.export_authorized_extensions = "tar.gz"
+
+      backuper = BackupRestore::Backuper.new(Discourse.system_user.id)
+      expect { backuper.send(:notify_user) }
+        .to change { Topic.private_messages.count }.by(1)
```

Comment:
Since the description of the test states that it includes the upload error, I think we should assert for the post's body instead of just checking a topic has been created.